x-pack/plugins/spaces/server/usage_collection/spaces_usage_collector.ts

```diff
@@ -179,50 +179,261 @@ export function getSpacesUsageCollector(
+        },
+        monitoring: {
+          type: 'long',
+          _meta: {
+            description: 'The number of spaces which have this feature disabled.',
+          },
+        },
+        graph: {
+          type: 'long',
+          _meta: {
+            description: 'The number of spaces which have this feature disabled.',
+          },
+        },
+        uptime: {
+          type: 'long',
+          _meta: {
+            description: 'The number of spaces which have this feature disabled.',
+          },
+        },
+        savedObjectsManagement: {
+          type: 'long',
+          _meta: {
+            description: 'The number of spaces which have this feature disabled.',
+          },
+        },
+        timelion: {
+          type: 'long',
+          _meta: {
+            description: 'The number of spaces which have this feature disabled.',
+          },
+        },
+        dev_tools: {
+          type: 'long',
+          _meta: {
+            description: 'The number of spaces which have this feature disabled.',
+          },
+        },
+        advancedSettings: {
+          type: 'long',
+          _meta: {
+            description: 'The number of spaces which have this feature disabled.',
+          },
+        },
+        infrastructure: {
+          type: 'long',
+          _meta: {
+            description: 'The number of spaces which have this feature disabled.',
+          },
+        },
+        visualize: {
+          type: 'long',
+          _meta: {
+            description: 'The number of spaces which have this feature disabled.',
+          },
+        },
+        logs: {
+          type: 'long',
+          _meta: {
+            description: 'The number of spaces which have this feature disabled.',
+          },
+        },
+        dashboard: {
+          type: 'long',
+          _meta: {
+            description: 'The number of spaces which have this feature disabled.',
+          },
+        },
+        ml: {
+          type: 'long',
+          _meta: {
+            description: 'The number of spaces which have this feature disabled.',
+          },
+        },
+        apm: {
+          type: 'long',
+          _meta: {
+            description: 'The number of spaces which have this feature disabled.',
+          },
+        },
+      },
+      available: {
+        type: 'boolean',
+        _meta: {
+          description: 'Indicates if the spaces feature is available in this installation.',
+        },
+      },
+      enabled: {
+        type: 'boolean',
+        _meta: {
+          description: 'Indicates if the spaces feature is enabled in this installation.',
+        },
+      },
+      count: {
+        type: 'long',
+        _meta: {
+          description: 'The number of spaces in this installation.',
+        },
+      },
+      'apiCalls.copySavedObjects.total': {
+        type: 'long',
+        _meta: {
+          description: 'The number of times the "Copy to Space" API has been called.',
```

Comment:
_[deep breath]_

This is a naming disparity that has bugged me for a while.

The module inside the Spaces plugin is called `copy_to_space`, and internally we refer to this as "copy (saved objects) (in)to space(s)" or "cts", but the HTTP route is `/api/spaces/_copy_saved_objects`. I think "Copy Saved Objects" is a better/more concise description and is more consistent with the other (Core) SO APIs, which is why I named these usage collector fields `apiCalls.copySavedObjects.*`.

So this is a minor nit, but how do you feel about changing this description to match as well?

Reply:
I'm on board, thanks for calling this out!